adoc/MAIN.release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+Release Notes: SUSE Cloud Application Platform 1.1
+==================================================
+
+:product: SUSE Cloud Applications Platform
+:version: 1.1
+:rn-url: https://www.suse.com/releasenotes
+:doc-url: https://www.suse.com/documentation/cloud-application-platform-1
+:deployment-url: https://www.suse.com/documentation/cloud-application-platform-1/book_cap_deployment/data/book_cap_deployment.html
+:suse: SUSE
+:caasp: {suse} Containers as a Service Platform
+:caaspa: {suse} CaaS Platform
+:ostack: OpenStack
+:cf: Cloud Foundry
+:scc: {suse} Customer Center
+:azure: Microsoft Azure
```

Comment:
As far as I can tell these variables are not expanded on GitHub. So the texts only can reasonably be read after the rendering step through DAPS, right?

Is there a way how the texts could be rendered in a human-readable way on GitHub already? That's what we have right now and it is useful to get early feedback and prepare notes before they end up in the official release notes or in the documentation.